patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -1505,6 +1505,30 @@
  			float num4 = (float)statLifeMax2 / 400f * 0.85f + 0.15f;
  			num2 *= num4;
  			lifeRegen += (int)Math.Round(num2);
+@@ -13069,8 +_,9 @@
+ 				manaRegen = statManaMax2 / 7 + 1 + manaRegenBonus;
+ 				if ((velocity.X == 0f && velocity.Y == 0f) || grappling[0] >= 0 || manaRegenBuff)
+ 					manaRegen += statManaMax2 / 2;
+-
++				
+ 				float num2 = (float)statMana / (float)statManaMax2 * 0.8f + 0.2f;
++
+ 				if (manaRegenBuff)
+ 					num2 = 1f;
+ 
+@@ -13080,7 +_,11 @@
+ 				manaRegen = 0;
+ 			}
```

Comment:
Note this is what was there before, diff just shows bc I originally had changes there, but took em out

Reply:
It's shown there because you or your editor added tabs to an empty line. Remove those and rediff.

Reply:
ty will do